client/src/components/DueDate/DueDate.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classes from './DueDate.module.css';
+
+const DueDate = props => {
+  const completeDate = new Date(props.date);
+  const months = [
```

Comment:
move this above the function and export it so that you can use this with tests also

Reply:
done